src/js/shot-generator/CameraControls.js

```diff
@@ -24,6 +24,8 @@ class CameraControls {
     window.addEventListener( 'pointerup', this.onPointerUp, false )
     window.addEventListener( 'keydown', this.onKeyDown, false )
     window.addEventListener( 'keyup', this.onKeyUp, false )
+
+    this.domElement.addEventListener('mousedown', this.onPointerDown, false )
```

Comment:
What is the `mousedown` for? Is `pointerdown` not catching something?

Reply:
I'll remove this. I was initially trying to use robotjs to click&drag a OrbitControl for camera rotation from phone, and the click from robotjs didn't seem to fire pointerdown event. But orbiting works different now so not needed